libs/shared/src/types/channel/index.ts

```diff
@@ -13,7 +13,6 @@ export enum StepTypeEnum {
   CHAT = 'chat',
   PUSH = 'push',
   DIGEST = 'digest',
-  TRIGGER = 'trigger',
```

Comment:
What's the reason to remove the TRIGGER step?

Reply:
We don't need these dummy jobs in new design, all dummy trigger jobs gone. The payload accumulation happening with _digestedNotificationIds of digest job.